env.example.yml

```diff
@@ -1,7 +1,9 @@
 LAMBDA_EXEC_SG: Insert AWS Security Group ID Here (it must be in the same VPC as the subnet)
 LAMBDA_EXEC_SUBNET: Insert AWS Subnet ID Here (it must be in the same VPC as the security group)
 BUGSNAG_NOTIFIER_KEY: INSERT BUGSNAG NOTIFIER KEY HERE
+HERE_API_KEY: INSERT HERE API KEY HERE
```

Comment:
`HERE_API_KEY` and `GEOCODE_EARTH_API_KEY` are never used at the same time, so can probably be combined into `API_KEY`, right?